src/content/structured/components/hero/code.mdx

```diff
@@ -300,12 +352,12 @@ export const snippetsImage = [
     subheading="Hero description. This is a Hero component, it should be used as a page heading."
     aligned="full-width"
   >
-    <IcButton variant="primary" slot="interaction">
+    <ic-button variant="primary" slot="interaction">
```

Comment:
I think these needed to stay as IcButton as the component previews use react components